ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py

```diff
@@ -155,19 +163,33 @@ def addtional_check(e2e_yaml_file, upgrade_yaml_file, missed_jobs):
     
 
 if __name__ == "__main__":
-    args = sys.argv
-    if len(args) < 3:
-        raise Exception("Missing e2e config file or upgrade config file")
-    e2e_yaml = args[1]
-    upgrade_yaml = args[2]
+    parser = argparse.ArgumentParser(description='Get missed upgrade jobs', 
+                                     add_help=True)
+    parser.add_argument("-e", "--e2e_yaml", type=str, required=True,
+                        help='the path of e2e job file')
+    parser.add_argument("-u", "--upgrade_yaml",type=str,
+                        help='the path of upgrade job file')
+    # upgrade mode, y means y version upgrade, z means z version upgrade
+    parser.add_argument("-m", "--mode", default="y", choices=["y", "z"], 
+                        help='the upgrade type, y: y stream upgrade, z: z stream upgrade')
+    # Only get e2e normal jobs which can be convert to upgrade
+    parser.add_argument("-j", "--jobs", action='store_true',
+                        help='A switch to indicate if only get e2e jobs')
     
-    e2e_jobs = get_jobs(e2e_yaml)
-    upgrade_jobs = get_jobs(upgrade_yaml)
+    args = parser.parse_args()
+    e2e_yaml = args.e2e_yaml
+    upgrade_yaml = args.upgrade_yaml
+    mode = args.mode
     
-    # upgrade mode, y means y version upgrade, z means z version upgrade
-    mode = args[3] if len(args) == 4 else 'y'
+    if args.jobs or not args.upgrade_yaml:
+        normal_jobs = get_jobs(e2e_yaml, mode=mode)
+        normal_jobs.sort()
+        print('\n'.join(normal_jobs))
+    else:
+        e2e_jobs = get_jobs(e2e_yaml, mode=mode)
+        upgrade_jobs = get_jobs(upgrade_yaml, mode=mode)
 
-    missed_jobs = get_missed_profiles(e2e_jobs, upgrade_jobs, mode=mode)
-    missed_jobs = addtional_check(e2e_yaml, upgrade_yaml, missed_jobs=missed_jobs)
-    missed_jobs.sort()
-    print('\n'.join(missed_jobs))
\ No newline at end of file
+        missed_jobs = get_missed_profiles(e2e_jobs, upgrade_jobs)
+        missed_jobs = addtional_check(e2e_yaml, upgrade_yaml, missed_jobs=missed_jobs)
+        missed_jobs.sort()
+        print('\n'.join(missed_jobs))
```

Comment:
nit: newline is missing after the last line in file so github complains with the red stop sign